event.go

```diff
@@ -400,9 +400,14 @@ var (
 // parseSimpleType wraps TdhFormatProperty to get rendered to string value of
 // @i-th event property.
 func (p *propertyParser) parseSimpleType(i int) (string, error) {
-	mapInfo, err := getMapInfo(p.record, p.info, i)
-	if err != nil {
-		return "", fmt.Errorf("failed to get map info; %w", err)
+	mapNameOffset := C.GetMapNameOffset(p.info, C.int(i))
+	var mapInfo unsafe.Pointer
+	if mapNameOffset != 0 {
```

Comment:
If we don't use the `mapNameOffset` variable later on, it can be omitted since the function name is already very descriptive.

```suggestion
	if C.GetMapNameOffset(p.info, C.int(i))!= 0 {
```